prow/cmd/deck/static/api/prow.ts

```diff
@@ -122,3 +122,10 @@ export interface ProwJobStatus {
   jenkins_build_id?: string;
   prev_report_states?: { [key: string]: ProwJobState };
 }
+
+// PodSpec is a description of a pod.
+// PodSpec mirrors the PodSpec struct defined in k8s.io/api/core/v1
+// Podspec interface only holds containers right now since no other values are used
+export interface PodSpec {
+  containers: object[];
+}
```

Comment:
I added this so that deck would know how many containers a job has. This way it'll know whether the build-log icon should lead to a single container build-log or to spyglass where users can see multiple. I'm not sure if I should add all of the other podspec fields to this just for consistency.